.kitchen.yml

```diff
@@ -70,7 +70,6 @@ suites:
     attributes:
       influxdb:
         install_type: file
-        version: 1.0.2
```

Comment:
this version is still required when specifying `install_type: file`.  ref: https://github.com/bdangit/chef-influxdb/blob/master/resources/install.rb#L53 . This should be okay to leave as 1.0.2.